lib/grpcServer/handlers/core/broadcastTransactionHandlerFactory.js

```diff
@@ -53,7 +53,7 @@ function broadcastTransactionHandlerFactory(insightAPI) {
 
     let transactionId;
     try {
-      transactionId = await insightAPI.sendTransaction(serializedTransaction.toString('hex'));
+      transactionId = await coreRPCClient.sendRawTransaction(serializedTransaction.toString('hex'));
     } catch (e) {
       if (e.statusCode === 400) {
```

Comment:
I guess with RPC you need to handle errors differently.

Reply:
Well, due to the rpc part in Dapi that erase the error code (only the message gets retained), I will remove this handling